### ISSUE TITLE
Docker build

### ISSUE THREAD

**Issue by talhazan** [jefc1111/zenbot-sim-runner#1]
Hello,

This tool seems amazing, would you mind sketch docker build? I can assist if you need.

The build beaks on my linux box, I haven't found any logs so debugging seems a bit complicated.

i would prefer to dockerize the app and continue from there.

Thank you!

**Comment by jefc1111** [comment]
Yes, good point. I definitely should set this up in Docker. I have a little Docker experience, but not a lot so it might take me some time. I will take a look in the next few days and comment here if I get stuck. Thanks for the offer of help!

Meanwhile, Laravel stores logs in `/storage/logs`. But I have a feeling you might be having problems before that point.

**Comment by jefc1111** [comment]
I have started a branch for this: https://github.com/jefc1111/zenbot-sim-runner/tree/dockerize

It's a work-in-progress (adapted from https://laravel-news.com/laravel-scheduler-queue-docker). The app works, with Redis and MySQL running, but I haven't yet worked out the best way to let it remote-control Zenbot. I could include a block for Zenbot in `docker-compose.yml` and take that route, but I was hoping to keep them more separate. Any ideas very welcome!

**Comment by holber0** [comment]
best way is when you use zenbot in a docker to. then add a network to both docker-compose files so they Docker Container can communicate. i will try this and let you know if this is working.

**Comment by jefc1111** [comment]
Ok, thanks. I haven't used Zenbot in Docker so not too familiar with that setup. I did a search on Stack Overflow etc on the subject of 'php app in Docker calling shell cmds in a different Docker container, or on host system' and all the solutions looked fairly involved. Do-able though... Meanwhile I have been working on giving the sim runner some self-learning capabilities. Fun! Let me know how you get on with the Docker stuff, happy to get involved.

**Comment by holber0** [comment]
didnt get it that you will use the zenbot binaray. so my idea is not working well. will try to build a new Docker environment with all together. 
PS: is it possible that your .env File still showing to your local mysql DB? because the "php artisan migrate" will not run in the container. php-mysql stuff is missing.

**Comment by jefc1111** [comment]
To get things working in Docker I had to set my db in `.env` like this;
`DB_HOST=mysql`
'mysql' being the name of the container running MySQL server.
